src/utils/hs_local.R

```diff
@@ -1,11 +1,11 @@
 #' Checks if this run is a test run
 #'
-#' `gmas_test_run()` looks for the environment variable `GMAS_TEST_RUN`. This is
+#' `hs_local()` looks for the environment variable `HS_LOCAL`. This is
 #' used in other functions so emails are not sent and data not saved back to the
 #' Google Cloud Storage bucket so test runs do not have any impact on the stored
```

Comment:
Edit text here to also reflect it's not about test runs (also not about Google Cloud Storage anymore ha), but just that all changes and outputs are local.

Reply:
yup good point